### DISCUSSION TITLE
Sfondo bianco su sezioni di input a partire da HA 2022.3

### DISCUSSION THREAD

**Discussion by manny199** [indomus/forum#1555] [GENERALE - Discussioni generiche]
Ciao a tutti,
dopo avere aggiornato alla nuova versione 2022.3, nonostante abbia selezionato e provato altri temi black, lo sfondo in alcune parti é bianco e non riesco a modificarlo (vedi screen shot).
C'è modo di farlo diventare uniforme con tutto il resto?
![immagine](https://user-images.githubusercontent.com/50017898/157514028-a8d08c56-ea27-4981-8998-dbfbbc40006c.png)

**Comment by KC-inDomus** [comment]
sì: adeguando il tema. apri una issue sul repository del tema e fai presente che va adeguato per MWC introdotto con la 2022.3.
